cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js

```diff
@@ -61,6 +62,8 @@ export default ({ service, variant, pageType, path }) => {
 
     getCookieBannerAcceptCanonical(service, variant).click();
 
+    cy.wait(1000);
```

Comment:
we only need to have a wait after the canonical accept click because that is the only time a request to the cookie oven is made

Reply:
Accepted your suggestions